web/web.go

```diff
@@ -99,10 +100,16 @@ func ConfigHandler(
 			shortLinkBaseURL = strings.TrimSuffix(cfg.BaseURL, "/") + "/l"
 		}
 
+		bypassGoogleAuth := false
```

Comment:
Maybe an explicit description of when the Google Auth would be skipped in a comment would help. Also, adding a note in the [configs/config.hcl](https://github.com/hashicorp-forge/hermes/blob/main/configs/config.hcl) file where Okta config is specified would help other users as well.

Reply:
Done!

Reply:
I don't see a note in the [configs/config.hcl](https://github.com/hashicorp-forge/hermes/blob/main/configs/config.hcl) file where Okta config is specified about Google auth being skipped.

Reply:
Ah, maybe I wasn't clear in my comments sorry about that. I was suggesting that we leave a note in form of a comment in the [configs/config.hcl](https://github.com/hashicorp-forge/hermes/blob/main/configs/config.hcl) file near the [Okta config block](https://github.com/hashicorp-forge/hermes/blob/main/configs/config.hcl#L131) to describe that when Okta is configured then Google Auth would be skipped. Does that make sense?

Reply:
Got it, updated https://github.com/hashicorp-forge/hermes/pull/144/commits/be6eb3f2973edeec9e0aa62d5db79ecc4264ca29